packages/react-dom/src/__tests__/ReactDOMServerSuspense-test.internal.js

```diff
@@ -105,39 +103,6 @@ describe('ReactDOMServerSuspense', () => {
     );
   });
 
-  // @gate experimental
-  it('server renders a SuspenseList component and its children', async () => {
-    const example = (
-      <React.SuspenseList>
-        <React.Suspense fallback="Loading A">
-          <div>A</div>
-        </React.Suspense>
-        <React.Suspense fallback="Loading B">
-          <div>B</div>
-        </React.Suspense>
-      </React.SuspenseList>
-    );
-    const element = await serverRender(example);
-    const parent = element.parentNode;
-    const divA = parent.children[0];
-    expect(divA.tagName).toBe('DIV');
-    expect(divA.textContent).toBe('A');
-    const divB = parent.children[1];
-    expect(divB.tagName).toBe('DIV');
-    expect(divB.textContent).toBe('B');
-
-    act(() => {
-      const root = ReactDOM.createBlockingRoot(parent, {hydrate: true});
```

Comment:
What's the reason you deleted this instead of switching it to `createRoot`?

Reply:
Moving too fast, fixed.